tests/dns.rs

```diff
@@ -0,0 +1,43 @@
+use std::{
+    path::PathBuf,
+    process::Command,
+    sync::mpsc::{self, Receiver, Sender},
+    thread,
+    time::Duration,
+};
+
+#[test]
+fn test_dns() {
+    let config_path = PathBuf::from("./.cargo/coinswap-test-data/directory/config.toml");
```

Comment:
I'm not sure, if we need `config_path` here, since we are using the default values to create `directoryd_process`